app/js/sagas/labOrdersSaga.js

```diff
@@ -149,7 +149,7 @@ export function* updateTestTypeFilter() {
 
 export function* fetchAndSetTestResultEncounter(args) {
   const state = yield select();
-  const { order, count = -1 } = args;
```

Comment:
I think is legacy from when the Lab Orders list had to fetch *all* the encounters associated with every order in the list, and can now be removed.